soaks/common/configs/tcp_blackhole.yaml

```diff
@@ -0,0 +1,3 @@
+worker_threads: 1
```

Comment:
Wondering if we should bump that number. I just copied it from the HTTP config:

https://github.com/vectordotdev/vector/blob/d4717f99910b15d7b688f645baf918fe41b0eff0/soaks/common/configs/http_blackhole.yaml#L1

There might be tests in the future that reuse the blackhole with multiple connections?